src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/FormatLoader/XmlFormatLoader10Test.php

```diff
@@ -132,11 +132,9 @@ public function testLoadDeprecated()
         $this->assertNotNull($result['3840x2160-retina']['internal']);
     }
 
-    /**
-     * @expectedException \Sulu\Bundle\MediaBundle\Media\FormatLoader\Exception\MissingScaleDimensionException
-     */
     public function testLoadDeprecatedWithMissingDimension()
     {
+        $this->expectException(\Sulu\Bundle\MediaBundle\Media\FormatLoader\Exception\MissingScaleDimensionException::class);
```

Comment:
According to our styleguides this class should actually be included via a `use` statement on top. And actually the same is true for all the `TestCase` classes from PHPUnit... Isn't there some StyleCI or PhpCsFixer rule we could apply that automatically fixes all of those for us?

Reply:
can not find anything about this :(

Reply:
did go manually over the `expectException` functions and fix it there